easybuild/easyconfigs/v/VTK/VTK-9.0.1-foss-2020b.eb

```diff
@@ -54,6 +54,7 @@ configopts += "-DOPENGL_gl_LIBRARY=$EBROOTMESA/lib/libGL.%s " % SHLIB_EXT
 configopts += "-DOPENGL_INCLUDE_DIR=$EBROOTMESA/include "
 # Python
 configopts += "-DVTK_WRAP_PYTHON=ON -DVTK_PYTHON_VERSION=3 -DVTK_PYTHON_OPTIONAL_LINK=OFF "
+configopts += '-DPython3_EXECUTABLE=$EBROOTPYTHON/bin/python '
```

Comment:
I tried
```
configopts += "-DPython3_ROOT_DIR=$EBROOTPYTHON "
```
which appears in later easyconfigs for VTK, but that didn't work

Reply:
I guess we should also try and check whether the `-DPython3_ROOT_DIR` is really correct, also when a newer system Python version is available too